Dockerfile

```diff
@@ -241,6 +246,7 @@ RUN \
 
 RUN \
   echo "**** DOWNLOAD AND INSTALL IPP ****" && \
+  no_proxy=https://registrationcenter-download.intel.com && \
```

Comment:
This is only valid for internal intel network and should be passed as a build parameter. This will brake install for all other proxied networks as it does to the runner currently.

Reply:
Ok, removed